sklearn/decomposition/_dict_learning.py

```diff
@@ -1557,6 +1595,9 @@ class DictionaryLearning(_BaseSparseCoding, BaseEstimator):
 
         .. versionadded:: 0.22
 
+    callback : callable, default=None
+        Callable that gets invoked every five iterations.
```

Comment:
We should add a `.. versionadded:: 1.2`

Reply:
let's move it above verbose and make the change in the signature of `__init__` as well